tools/CoqMakefile.in

```diff
@@ -616,9 +616,9 @@ install-extra::
 	@# Extension point
 .PHONY: install install-extra
 
-META: $(METAFILE)
+META: "$(METAFILE)"
```

Comment:
This is a misconception. Quotes do not play any specific role in a Makefile; they are regular characters that are part of file names. If `$(METAFILE)` were to contain spaces, then they would need to be prefixed by backslashes to be protected.

Reply:
```suggestion
META: $(METAFILE)
```